cluster/gce/coreos/master.yaml

```diff
@@ -135,6 +135,7 @@ coreos:
         --reconcile-cidr=false
         Restart=always
         RestartSec=10
+        KillMode=process
```

Comment:
In the master node, excepting the kubelet, all other components are expected to run in containers. So this change should be fine.